### ISSUE TITLE
Does not show new mail in global inbox folder (nor in junk one)

### ISSUE THREAD

**Issue by MicioMax** [gyunaev/birdtray#17]
I added this account : 

mailbox://nobody@Local%20Folders/Inbox

The corresponding path on my pc is : 

/home/massimo/.thunderbird/xhhp31gc.default/Mail/Local Folders/Inbox
(and Inbox.msf)

When I installed it it showed some non-existing new mails; i fixed with Fix button, now the icon is ok, but new mails are not shown.
The rest is more than perfect, I was waiting since longtime for such an extension.

**Comment by JackDinn** [comment]
yea, i tested the unified inbox to and it does not seem to work but i only have 7 accounts so no big problem to do them individually for me.

**Comment by JackDinn** [comment]
Assuming that  `/home/greg/.thunderbird/tzvg3gbn.default/Mail/smart mailboxes/Inbox.msf` is correct for the unified Inbox i can not get it to work. It shows 96 new mail, it can be "fixed" but still does not register any new mail. (could it be the un-escapped space in the path).

The ImapMail folders work fine individually.

**Comment by gyunaev** [comment]
What does it show in the console?

**Comment by JackDinn** [comment]
not a lot really.
```
❰greg❙~/git/birdtray/src(git✱≠master)❱✘≻ ./birdtray
Window ID found: 4C00011
unreadCounterUpdate -1

```
 Its stuck on 96 now though i cant "fix" it and it does not change with unread mail or marking mail read.

**Comment by JackDinn** [comment]
The only place iv got 96 in my unified Inbox is
`(96=msgCharSet)`

**Comment by gyunaev** [comment]
How is this "global inbox" set up? View-Folders-Unified is just a view, it doesn't change anything.

**Comment by JackDinn** [comment]
o so you are saying that we still need to add all the individual ImapMail/ inboxes and can't just add the single unified inbox ? well thats fine i just thought that instead of adding them individually i might be able to add just add the unified inbox which lists all of my inboxes :-
/home/greg/.thunderbird/tzvg3gbn.default/Mail/smart mailboxes/Inbox.msf

**Comment by gyunaev** [comment]
I don't know as I cannot reproduce the "smart mailboxes" scenario. If I enable "Unified view" it does not create any new mailbox files, so its just a view built in software.

Can you please do the following:

- Have all messages read;
- Copy this Inbox.msf to Inbox.msf.0
- Make one email unread;
- Copy this Inbox.msf to Inbox.msf.1
- diff -u both and see if there's any difference?

**Comment by JackDinn** [comment]
k
```
g❙~/.thunderbird/tzvg3gbn.default/Mail/smart mailboxes❱✔≻ diff -u Inbox.msf.0 Inbox.msf.1
--- Inbox.msf.0 2018-10-05 08:08:31.188533754 +0100
+++ Inbox.msf.1 2018-10-05 08:08:43.061743971 +0100
@@ -75,3 +75,7 @@
 @$${15B{@
 [1:^9F(^A2=0)]
 @$$}15B}@
+
+@$${15C{@
+[1:^9F(^A2=1)]
+@$$}15C}@

```

**Comment by JackDinn** [comment]
it says in the unified Inbox.msf 
(A2=numNewMsgs)

**Comment by gyunaev** [comment]
If you can create one of those MSFs which you can privately share with me, one with 0 unread and one with 1 unread? gyunaev@ulduzsoft.com

**Comment by gyunaev** [comment]
> it says in the unified Inbox.msf
> (A2=numNewMsgs)

Now I need to find out the table index for it. Its not part of the any MSF I have.

**Comment by gyunaev** [comment]
Should be fixed by 322ede1

**Comment by JackDinn** [comment]
yep, thats working. I only need to add the one unified Inbox.msf now and that works for all my inbox's.

I would assume that will apply to OP's "Local Folders" to.

Nice work, Cheers.
